dds_web/forms.py

```diff
@@ -76,10 +73,47 @@ class LogoutForm(flask_wtf.FlaskForm):
     logout = wtforms.SubmitField("Logout")
 
 
+# TODO: Remove TwoFactorAuthForm and connected endpoints.
 class TwoFactorAuthForm(flask_wtf.FlaskForm):
     secret = wtforms.HiddenField("secret", id="secret")
     otp = wtforms.StringField(
         "otp",
         validators=[wtforms.validators.InputRequired(), wtforms.validators.Length(min=6, max=6)],
     )
     submit = wtforms.SubmitField("Authenticate User")
+
+
+class RequestResetForm(flask_wtf.FlaskForm):
+    """Form for attempting password reset."""
+
+    email = wtforms.StringField(
+        "Email",
+        validators=[
+            wtforms.validators.DataRequired(),
+            wtforms.validators.Email(),
+            utils.email_taken_wtforms(),
+        ],
+    )
+    submit = wtforms.SubmitField("Request Password Reset")
+
+
+class ResetPasswordForm(flask_wtf.FlaskForm):
+    """Form for setting a new password."""
+
+    password = wtforms.PasswordField(
+        "Password",
+        validators=[
+            wtforms.validators.DataRequired(),
+            wtforms.validators.EqualTo("confirm", message="Passwords must match!"),
```

Comment:
`"confirm"` here leads to an error when attempting to reset password since the field is still called `confirm_password`. Change either here or the field and I think it should work